openff/toolkit/tests/create_molecules.py

```diff
@@ -18,8 +18,11 @@
 
 import numpy as np
 from simtk import unit
+import pytest
```

Comment:
I wonder if the existing molecules should be fixtures as well?

Reply:
I think they should all become fixtures, and I had to stop myself from converting them all over so that this PR wouldn't get absolutely humongous.

Reply:
Sounds good - agree it's best to leave it be in this scope